src/services/websocket.js

```diff
@@ -0,0 +1,31 @@
+import config from '../config';
+
+const CONNECTION_ESTABLISHED_EVENT = 'ws-connection-establish';
+
+let ws = new WebSocket(config.api.websocketEndpoint);
+
+export default function initializeWebSocket(onMessage) {
+  connectWebSocket(onMessage);
+}
+
+/**
+ * Connect to the websocket endpoint
+ *
+ * @param onMessage
+ */
+function connectWebSocket(onMessage) {
+  ws.onclose = function () {
```

Comment:
Or, we can extract these two functions out of the `connectWebSocket` function. As they may grow later.

Something like this:
```js
function connectWebSocket(onMessage) {
  ws.onclose = handleClose;
  ws.onmessage = handleMessage;

  // Other code here.
  ...
}

/**
 * Handles websocket connection close.
 */
function handleClose(e) {
  ...
}

/**
 * Handles websocket incomming message .
 */
function handleMessage(e) {
  ...
}
```